config/routes.rb

```diff
@@ -1,10 +1,13 @@
 Rails.application.routes.draw do
   devise_for :users
   resources :cars do
+  resources :likes, only: [:create, :destroy]
+  resources :comments
     collection do
       get 'my_cars', to: 'cars#my_cars'
     end
   end
+  resources :likes, only: [:create, :destroy]
```

Comment:
I'm confused why this route is duplicate

Reply:
Sorry, my bad. I can resolve it.